### PR TITLE
Create API for addonsConfigurations

### PR DESCRIPTION
**Pull request by polskikiel**
**Description**

Changes proposed in this pull request:

- Create API for `addonsConfigurations` - the config maps defined in kyma-system with label `helm-broker-repo=true`, which defines the bundles' repositories.

**Related issue(s)**

### CODE REVIEW

File: components/console-backend-service/before-commit.sh [hunk 1]
```diff
@@ -93,14 +93,14 @@ if [[ ${buildErrCheckResult} != 0 ]]; then
...
-else echo -e "${GREEN}√ errcheck ${NC}"
-fi
+#errCheckResult=$(./errcheck-vendored -blank -asserts -ignoregenerated ./...)
+#rm errcheck-vendored
+#
+#if [[ $(echo ${#errCheckResult}) != 0 ]]; then
+#    echo -e "${RED}✗ [errcheck] unchecked error in:${NC}\n${errCheckResult}${NC}"
+#    exit 1
+#else echo -e "${GREEN}√ errcheck ${NC}"
+#fi
```
Comment (piotrmiskiewicz) [inline components/console-backend-service/before-commit.sh]: decide what shoudl be done here - uncomment or remove it

File: components/console-backend-service/internal/domain/servicecatalog/addonsconfiguration_service.go [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+	}
+
+	return result, nil
+}
+
+func (s *addonsConfigurationService) setLabels(givenLabels *gqlschema.Labels) map[string]string {
+	labels := map[string]string{
+		addonsCfgLabelKey: addonsCfgLabelValue,
+	}
+	if givenLabels != nil {
```
[the remark this answers is not in the document]
Comment (piotrmiskiewicz) [reply components/console-backend-service/internal/domain/servicecatalog/addonsconfiguration_service.go]: discussed

File: components/console-backend-service/before-commit.sh (hunk 1 above)
Comment (polskikiel) [reply components/console-backend-service/before-commit.sh]: autogenerated files need this line at the beginning: `// Code generated by mockery v1.0.0. DO NOT EDIT.` to skip them in this check... uncommented

**Pull request merged**